src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -136,11 +143,20 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
               style={{ width: inputWidth }}
               role="input"
             />
-          </>
-        ) : (
-          renderTypography({ value: inputValue, className: styles.typograpy, ...clickableProps })
-        )}
-      </div>
+          )}
+          <AnimatedTypography
+            ref={typographyRef}
+            aria-hidden={isEditing}
```

Comment:
When it's in edit mode I "hide" it but keeping the width to calculate the width of the input